### DISCUSSION TITLE
How can i use multi db environment?

### DISCUSSION THREAD

**Discussion by breeezy96** [Intechnity-com/OdooJsonRpcClient#16] [General]
Hello. With one database it works, but when I create another one, I can't login because it says: The requested URL was not found. What else should I type in the URL? I heard about cookies, but I didn't see any attribute for that.

Thanks in advance,

Máté

**Comment by patricoos** [comment]
What version of odoo do U use? as far as I know the url changes with the database
```
new OdooConfig(
            apiUrl: "https://db-name.dev.odoo.com",
            dbName: "db-name",
            userName: "admin",
            password: "admin"
        );
```

**Reply by breeezy96** [reply]
My URL looks like this: "http://odoo11.wib.hu:port". It doesn't change with the database...

**Reply by patricoos** [reply]
I test it several times on my localhost odoo instance and everything is working correctly. Check your configuration.
